src/components/standup/KanbanGroupSection.tsx

```
@@ -120,15 +134,21 @@ export default function KanbanGroupSection({
+  // to sync after a refresh and to roll back a drag that didn't stick.
+  const syncFromProps = useCallback(() => {
     const map: Record<string, StandupWorkItem[]> = {};
     for (const col of columns) {
       map[col.name] = col.items;
     }
     setLocalItems(map);
   }, [columns]);
 
+  // Sync with prop changes (e.g. after refresh)
+  useEffect(() => {
+    syncFromProps();
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/components/standup/KanbanGroupSection.tsx:149:5
  147 |   // Sync with prop changes (e.g. after refresh)
  148 |   useEffect(() => {
> 149 |     syncFromProps();
      |     ^^^^^^^^^^^^^ Avoid calling setState() directly within an effect
  150 |   }, [syncFromProps]);
  151 |
  152 |   const sensors = useSensors(

(react-hooks/set-state-in-effect)